hackathon_site/dashboard/frontend/src/components/orders/OrdersFilter/OrderFilter.tsx

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { Formik, Field, FieldProps, FormikValues } from "formik";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Chip from "@material-ui/core/Chip";
+import Paper from "@material-ui/core/Paper";
+import Divider from "@material-ui/core/Divider";
+import Checkbox from "@material-ui/core/Checkbox";
+import Radio from "@material-ui/core/Radio";
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormGroup from "@material-ui/core/FormGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import { OrdersOrdering, OrderStatus } from "api/types";
+import styles from "../../global_css/Filter.module.scss";
+
+type OrderByOptions = {
+    value: OrdersOrdering;
+    label: string;
+}[];
+
+type StatusCategories = [
+    {
+        id: Number;
+        status: OrderStatus;
+        numOrders: Number;
+    }
+];
+
+const orderByOptions: OrderByOptions = [
+    { value: "", label: "Default" },
+    { value: "timeOrderedASC", label: "Time Ordered (ASC)" },
+    { value: "timeOrderedDESC", label: "Time Ordered (DESC)" },
+    { value: "OrderQuality", label: "Order Quantity" },
+];
+
+const RadioOrderBy = ({ field, options }: FieldProps & { options: OrderByOptions }) => (
+    <RadioGroup {...field} name={field.name}>
+        {options.map((item, i) => (
+            <FormControlLabel
+                name={field.name}
+                value={item.value}
+                label={item.label}
+                control={<Radio color="primary" />}
+                checked={field.value === item.value}
+                key={i}
+            />
+        ))}
+    </RadioGroup>
+);
+
+const CheckboxCategory = ({
+    field,
+    options,
+}: FieldProps & { options: StatusCategories }) => (
+    <FormGroup {...field}>
+        {options.map((item, i) => (
+            <div className={styles.filterCategory} key={i}>
+                <FormControlLabel
+                    name={field.name}
+                    value={item.id}
+                    control={<Checkbox color="primary" />}
+                    label={item.status}
+                    checked={
+                        field.value ? field.value.includes(item.id.toString()) : false
+                    }
+                />
+                <Chip
+                    size="small"
+                    label={item.numOrders}
+                    className={styles.filterCategoryChip}
+                />
+            </div>
+        ))}
+    </FormGroup>
+);
+
+const OrderFilter = ({ handleReset, handleSubmit }: FormikValues) => {
+    const status = [
+        {
+            id: 1,
```

Comment:
you don't need id field here because we'll be passing `status` to the api

Reply:
Because you're removing id, you should replace the usage of id in CheckboxCategory component with status, since status should be the unique identifier

Reply:
also, could you move this status object to mockData.tsx because you can use this for tests in the future, but the actual statuses will be returned from our api eventually.